UM/Qt/QtApplication.py

```diff
@@ -186,6 +186,12 @@ def initialize(self) -> None:
         Logger.log("i", "Initializing version upgrade manager ...")
         self._version_upgrade_manager = VersionUpgradeManager(self)
 
+    def _displayLoadingPluginSplashMessage(self, plugin_id: Optional[str]) -> None:
+        message = i18nCatalog("uranium").i18nc("@info:progress", "Loading plugins...")
+        if plugin_id:
+            message = f"{i18nCatalog('uranium').i18nc('@info:progress', 'Loading plugin')} {plugin_id}..."
```

Comment:
Again; i'm not really sure if i'm a fan of this way of getting handling the translations.

I personally think that `i18nCatalog('uranium').i18nc('@info:progress', 'Loading plugin {plugin_id}').format(plugin_id=plugin_id)` is much more readable.